app/Http/Controllers/BeatmapsController.php

```diff
@@ -52,29 +53,55 @@ public function scores($id)
 
         $beatmap = Beatmap::findOrFail($id);
         $mode = Request::input('mode', Beatmap::modeStr($beatmap->playmode));
+        $enabled_mods = Request::input('enabledMods');
```

Comment:
pass in array instead and convert to bitset by extending function (convert to library?) used for the other direction.